companion.php

```diff
@@ -86,6 +86,10 @@ function companion_admin_notices() {
 			<strong>Server path:</strong> <code id="jurassic_ninja_server_path" class="jurassic_ninja_field"><?php echo esc_html( get_home_path() ); ?></code>
 			<?php clipboard( 'jurassic_ninja_server_path' ); ?>
 		</p>
+		<p>
+			<strong>Jetpack rsync path:</strong> <code id="jurassic_ninja_rsync_path" class="jurassic_ninja_field"><?php echo esc_html( $sysuser ) . '@' . esc_html( str_replace( 'https://', '', network_site_url() ) ) . ':' . esc_html( get_home_path() ) . 'wp-content/plugins' ; ?></code>
```

Comment:
I like that idea! What about adding the full jetpack rsync command?

```suggestion
			<strong>Jetpack rsync path:</strong> <code id="jurassic_ninja_rsync_path" class="jurassic_ninja_field">jetpack rsync jetpack <?php echo esc_html( $sysuser ) . '@' . esc_html( str_replace( 'https://', '', network_site_url() ) ) . ':' . esc_html( get_home_path() ) . 'wp-content/plugins/jetpack' ; ?></code>
```